### PR TITLE
Feature/simplify create your first sgx app

### PR DESCRIPTION
**Pull request by jbern0rd**

### CODE REVIEW

File: for-developers/confidential-computing/create-your-first-sgx-app.md [hunk 1]
```diff
@@ -21,7 +21,10 @@ The short answer is: the application is protected by taking a snapshot of the fi
 
 ## Prepare your application
 
-Create a directory tree for your application in `~/iexec-projects/`.
+For this tutorial, you will start from the end of [Your first application](../your-first-app.md) tutorial and your `<dockerusername>/my-hello-world:1.0.0` OCI image.
```
Comment (james-toussaint) [inline for-developers/confidential-computing/create-your-first-sgx-app.md]:
```suggestion
Before going further, your `<dockerusername>/my-hello-world:1.0.0` image built previously is required. If you missed that part, please go back to [Your first application](../your-first-app.md).
```

File: for-developers/confidential-computing/create-your-first-sgx-app.md [hunk 2]
```diff
@@ -169,15 +72,9 @@ ENTRYPOINT="node /app/app.js"
 # declare an image name
 IMG_NAME=tee-hello-world
```
Comment (james-toussaint) [inline for-developers/confidential-computing/create-your-first-sgx-app.md]: I think we should update to:
`IMG_NAME=<dockerusername>/tee-hello-world`
plus references such as:
`docker run --rm -e SCONE_HASH=1 <dockerusername>/tee-hello-world:tee-debug`

File: for-developers/confidential-computing/create-your-first-sgx-app.md (hunk 1 above)
Comment (jbern0rd) [reply for-developers/confidential-computing/create-your-first-sgx-app.md]: see 906e9c9

File: for-developers/confidential-computing/create-your-first-sgx-app.md (hunk 2 above)
Comment (jbern0rd) [reply for-developers/confidential-computing/create-your-first-sgx-app.md]: in documentation, `IMG_NAME` is used for `--name` option of sconifier.
To keep things consistent with our examples, I went for
```
IMG_NAME=tee-hello-world
IMG_TO=<dockerusername>/${IMG_NAME}:tee-debug

    --name=${IMG_NAME} \
    --from=${IMG_FROM} \
    --to=${IMG_TO} \
```

I kept `IMG_TO` in mrenclave retrieval `docker run` command and updated the last command with the full image name.
See 69a9c83 and 868bf35

Did I miss something ?

**Review by james-toussaint — APPROVED** [review]

**Pull request merged**